Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -346,10 +354,16 @@ private fun PlayerContentRegular(
     onBackPress: () -> Unit,
     onAddToQueue: () -> Unit,
     playerControlActions: PlayerControlActions,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
 ) {
     val playerEpisode = uiState.episodePlayerState
     val currentEpisode = playerEpisode.currentEpisode ?: return
+
+    val sharedTransitionScope = LocalSharedTransitionScope.current
```

Comment:
Could this also be done in the non-regular `PlayerContent`s for getting the same shared element transition for tabletop mode? Or maybe this could be done within the `PlayerImage` itself to avoid duplication?